definitions.yaml

```diff
@@ -404,6 +409,20 @@ definitions:
                 type: string
                 description: "A string representing the status of the app. (Can be: empty, DEPLOYED or FAILED)"
 
+  V4AppSpecUserConfigConfigMap:
+    type: object
+    description: User definable configuration to be applied when the app is deployed
+    properties:
+      config_map:
```

Comment:
The name `config_map` suggests that this represents an entire Kubernetes configmap. My understanding was that this only represents the `data.values` part. So maybe name it `configmap_values` or `configmap_data_values`?

Reply:
Oh, strike that. I was in a different thread mentally.

As this seems to reference a `ConfigMap` name, in general the name is fine. My only issue is that I'd rather translate it to `configmap` than `config_map`.